src/services/phrases.service.ts

```diff
@@ -44,10 +45,32 @@ const getPhrasesId = (idPhrases: IdPhrases) => {
   });
 };
 
+const getPolynomialPhrases = async (polynomialName: string) => {
+  let phraseId: string;
+  const polynomial = await Polynomial.findAll({
+    attributes: ['id', 'name'],
+  });
+
+  const dataArray = polynomial.map(result => ({
+    id: result.getDataValue('id'),
+    name: result.getDataValue('name'),
+  }));
+
+  for (let i = 0; i < dataArray.length; i++) {
+    if (dataArray[i].name === polynomialName) {
+      phraseId = dataArray[i].id;
+      return await Phrases.findAll({
+        where: { polynomial_id: phraseId },
+      });
+    }
+  }
+};
+
```

Comment:
creo que lo mejor sería realizar la búsqueda directa con el id, o agregar la relación en frases para buscar a través del nombre. Utilizar dos findAll y un for no lo veo eficiente